apps/sms/views/inbox/js/inbox.js

```diff
@@ -163,8 +164,7 @@ var InboxView = {
     var threadId = node.dataset.threadId;
     var draftId = node.dataset.draftId;
 
-    var threadOrDraft = draftId ?
-      Drafts.byDraftId(+draftId) : Threads.get(+threadId);
+    var threadOrDraft = Threads.get(+(draftId || threadId));
```

Comment:
Right, I just want to have the same interface for "getConversationSummary" result that is going to be used in ConversationView where full draft is needed. Later if it becomes issue for us in Inbox, we can split ConversationSummary to two types, but didn't think about this too much yet.